quickstarts/mlops/aporia/dashboards/aporia.json

```diff
@@ -0,0 +1,183 @@
+{
+  "name": "Aporia Dashboard",
+  "description": "Aporia predictions monitoring dashboard",
+  "permissions": "PUBLIC_READ_WRITE",
```

Comment:
```suggestion
```

This line will also need to be removed, this is why it's failing the schema check